packages/lit-html/package.json

```diff
@@ -402,7 +403,8 @@
         "test:prod",
         "test:node",
         "test:node-dev",
-        "check-version"
+        "check-version",
+        "test:prodsize"
```

Comment:
In `lit-html` you have this run as part of the tests, but I don't see that with `lit`.

I think it might be best to have these run as part of the benchmark workflow, and instead of being run by Wireit as part of another step, have it in it's own step so if it fails we see it separated in the GH workflow logs.

Being part of the benchmark runner would also let us compare sizes against head and the latest release similarly to the benchmarks, since we already check those out and build them.

Reply:
Refactored to be part of the benchmark runner and to report as comments on the PR.

I'd maybe have liked it if there was no comment in the case there's no size change, but this way we do know that the size benchmark at least, and it ensures that we can update the comment after an initial comment reporting a problem.